styles.css

```diff
@@ -533,3 +533,117 @@ button[enabled] {
   border-radius: 150px;
   margin-right: 12px;
 }
+
+.contact-form,
+.my-form {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 12px;
+  background-color: #e5e5e5;
+}
+
+.contact-form-content {
+  margin-top: 108px;
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  margin-bottom: 70px;
+}
+
+.contact-form-name,
+.contact-form-email {
+  font-family: 'Inter', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  color: #979493;
+  border: none;
+  height: 18px;
+  border-bottom: 1px solid #dbd8d7;
+  padding: 20px;
+  margin-bottom: 24px;
+}
+
+.contact-form-message {
+  margin-bottom: 24px;
+  border: none;
+  padding: 20px;
+  background-color: #fbf8f7;
+}
```

Comment:
- [ ] Kindly adjust the width of the `textarea` in order to meet the provided design. :wink: 

   - ### Your design
  
   ![image](https://user-images.githubusercontent.com/87197412/186627615-15fd4e5c-0414-416e-ba71-42ef6d4c783c.png)


    - ### Figma design
   ![image](https://user-images.githubusercontent.com/87197412/186627802-098cf614-05d2-44a3-8a6a-1b298505960b.png)